src/server/util/kubectl.cc

```diff
@@ -101,6 +117,14 @@ static std::string generate_global_object(
                     "\nkind: GlobalObject"
                     "\nmetadata:"
                     "\n  name: " + object_id +
+                    "\n  namespace: " + namespace_ +
+                    "\n  labels:"
+                    "\n    job: none"
+                    "\n  ownerReferences:"
+                    "\n    - apiVersion: v1"
+                    "\n      kind: Pod"
+                    "\n      name: " + pod_name +
+                    "\n      uid: " + uid +
```

Comment:
I was imaging the owner reference of those local objects / global objects should be the `Vineyardd` deployment, rather than some single pod.

Because if one node failure, other node will do some garbage collection to deleted the local objects that been placed on that node, as well as global objects that requires those local objects. During this process those CRDs will be deleted by vineyardd.

We want to the owner reference to handle cases that when the vineyardd cluster (vineyardd CRDs) been deleted as a whole, there would be no role to do the garbage collection work, thus we need to leverage the owernship reference to remove those local / global object CRDs.

Reply:
Yep, in such a scene, using the pod as ownerreference is unsuitable. I will add a `kubectl get` API  to query the `vineyardd` deployment in the `kubectl.cc`.

Reply:
No need to such an API.

If those vineyardd pod is managed by a `Vineyardd` CRD, you could add the UID of `Vineyardd` CRD to the environment variable of those pods, I think.